include/osquery/query.h

```diff
@@ -22,6 +22,16 @@
 
 namespace osquery {
 
+enum ColumnType {
+  UNKNOWN_TYPE = 0,
+  TEXT_TYPE,
+  INTEGER_TYPE,
+  BIGINT_TYPE,
+  UNSIGNED_BIGINT_TYPE,
+  DOUBLE_TYPE,
+  BLOB_TYPE,
```

Comment:
Seems reasonable, but for this change, I'm just moving this enum here, as is, from tables.h so that it can be both used as it is by the the code that determines which of these types it is, and also by the new serialization code that chooses how to serialize a field based on the specified type.